Sources/MapboxMaps/Foundation/Extensions/Bundle+MapboxMaps.swift

```diff
@@ -10,4 +14,13 @@ extension Bundle {
         return Bundle(for: BundleLocator.self)
         #endif
     }
+
+    static var mapboxMapsMetadata: MapboxMapsMetaData? = {
+        guard let metadataPath = Bundle.mapboxMaps.url(forResource: "MapboxMaps", withExtension: "json"),
+              let data = try? Data(contentsOf: metadataPath) else {
+            return nil
+        }
+
+        return try? JSONDecoder().decode(MapboxMapsMetaData.self, from: data)
+    }()
```

Comment:
Can we assume that this will never fail and have it return a non-optional? Seems like a unit test should provide all the safety we need.